cypress/integration/DeleteTest.ts

```diff
@@ -0,0 +1,85 @@
+describe("Delete Test", () => {
+  it("Instance", () => {
+    cy.visit("/");
+
+    cy.ouiaId("Instances list table", "PF4/Table")
+      .ouiaId("Instance ten", "PF4/TableRow")
+      .find("td")
+      .then(($cells) => {
+        expect($cells).have.length(4);
+        cy.wrap($cells.eq(3)).ouiaType("PF4/Dropdown").click();
```

Comment:
Just information comment:
Maybe we should decide which approach we will use. Yours pure cypress solution or this approach https://github.com/jomarko/sandbox-ui/blob/MGDOBR-676%2C677/cypress/integration/InstancesTest.ts#L36-L37. Where I mix CSS selector.

I am not sure if there is necessary to assert the length. I never use it but the another pure cypress solution could be `.find("td").eq(3).ouiaType("PF4/Dropdown").click();`

Reply:
I do not think there is a simple answer what approach is the best. Speaking about case that `get`/`find` returned collection of elements.

I think it depends if we are interested in single element of returned collection vs in multiple elements of given collection.

For the first case I use , after your review comments, approach like this:
```
...
        .find("td")
        .eq(1)
        .should("have.text", "deprovision");
```

for the second case:
```
...
        .find("td")
        .then(($cells) => {
          expect($cells).have.length(6);
          expect($cells.eq(0)).have.text("Processor one");
          expect($cells.eq(3)).have.text("ready");
          cy.wrap($cells.eq(5)).ouiaType("PF4/Dropdown").click();
        });
```